pygfx/utils/color.py

```diff
@@ -255,6 +255,26 @@ def css(self):
         else:
             return f"rgba({int(255*r+0.5)},{int(255*g+0.5)},{int(255*b+0.5)},{a:0.3f})"
 
+    def multiply_scalar(self, factor):
```

Comment:
> The reason why this function was added before is that in the constructor of Color, the rgba component is truncated to a maximum of 1.0.

I'd be open to removing clipping of values here, and apply clipping in the shader instead.